policy-controller/k8s/index/src/lib.rs

```diff
@@ -193,3 +196,31 @@ impl Index {
         }
     }
 }
+
+// === impl Errors ===
+
+impl std::fmt::Display for Errors {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0[0])?;
+        for e in &self.0[1..] {
+            write!(f, "; and {}", e)?;
+        }
+        Ok(())
+    }
+}
+
+impl std::error::Error for Errors {
```

Comment:
i was wondering if it made sense to somehow override downcasting to any error in the vec...but it's probably not worth it, since these errors are all basically untyped strings and we never inspect them programmatically...